### ISSUE TITLE
bug: ModuleNotFoundError: No module named 'openllm_client'

### ISSUE THREAD

**Issue by crclark** [bentoml/OpenLLM#194]
### Describe the bug

Tried to follow the basic example of the client in the readme.

### To reproduce

```bash
conda create -n openllm-test
conda activate openllm-test
pip install openllm==0.2.20
```

```python
import openllm
client = openllm.client.HTTPClient('http://localhost:3000')
```

raises:

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/connorclark/.local/lib/python3.9/site-packages/openllm/client.py", line 48, in __getattr__
    return getattr(importlib.import_module(f".{module}", _module), name)
  File "/usr/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 984, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'openllm_client'
```

### Environment

see above

**Comment by aarnphm** [comment]
What platform is this?

**Comment by crclark** [comment]
> What platform is this?

I've reproduced this on

- Debian on GCP with GCP's deep learning VM image (image project= `deeplearning-platform-release` and image name = `common-gpu-v20230615-debian-11-py310`).
- OpenSUSE Tumbleweed on my personal desktop.
- MacOS Ventura on my work laptop.

**Comment by Sambonic** [comment]
Same issue here. I ran it on vscode and I am on a windows OS.

**Comment by aarnphm** [comment]
I'm working on a new release soon.

**Comment by aarnphm** [comment]
This has been fixed with 0.2.22.
